### ISSUE TITLE
Feature: Screencast

### ISSUE THREAD

**Issue by monsieurpouet** [keredson/gnomecast#148]
Hi ! 

It's possible to add a feature about screencasting from gnomecast ?
It seems work with ffmpeg :+1: 

https://trac.ffmpeg.org/wiki/Capture/Desktop

:kissing:

**Comment by keredson** [comment]
Probably.  But chrome already does this well, doesn't it?

**Comment by monsieurpouet** [comment]
Yes it is ! You right, but if I preferred not to use Chrome is I can.
